lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +36,137 @@ fields:
     example: |
       {
         "id": "SH-12508041"
-      }
\ No newline at end of file
+      }
+
+  - name: questionnaire
+    required: 'Yes'
+    type: Questionnaire
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire
+    description: The Questionnaire that defines and organizes the questions for which answers are being provided.
+    action: update
+    example: |
+      {
+        "questionnaire": "https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041"
+      }
+
+  - name: status
+    required: 'Yes'
+    type: code
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.status
+    description: The position of the questionnaire response within its overall lifecycle.
+    action: update
+    example: |
+      {
+        "status": "in-progress"
+      }
+    note: status must be 'in-progress'
+  
+  - name: subject
+    required: 'Yes'
+    type: Reference
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.subject
+    description: The subject of the questionnaire response. This could be a patient, organization, practitioner, device, etc. This is who/what the answers apply to, but is not necessarily the source of information.
+    action: update
+    example: |
+      {
+        "subject": {
+          "reference": "Patient/12508041"
+        }
+      }
+
+  - name: item
+    required: 'Yes'
+    type: List of BackboneElement
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item
+    description: A group or question item from the original questionnaire for which answers are provided. Only answer value types of String, Quantity, and Coding are supported.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93",
+            "item": [
+              {
+                "linkId": "93-123",
+                "answer": [
+                  {
+                    "valueQuantity": {
+                      "value": 20
+                    }
+                  }
+                ]
+              }
+            ] 
+          }
+        ]
+      }
+
+  - name: item.linkId
+    required: 'Yes'
+    type: string
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId
+    description: The item from the Questionnaire that corresponds to this item in the QuestionnaireResponse resource.
+    action: update
+    example: |
+      {
+        "linkId": "93"
+      }
+
+  - name: item.item
+    required: 'No'
+    type: List of BackboneElement
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.item
+    description: Questions or sub-groups nested beneath a question or group.
```

Comment:
For us, item.items are just Questions nested beneath a group.

Reply:
b21bdddcfadec00f9bd0f1374e4268810bc889f8